### PR TITLE
Major updates

### PR DESCRIPTION
**Pull request by myugan**
Move some resources and remove unrelated resources

### CODE REVIEW

**Comment by myugan** [comment]
Fixing merge conflict

**Comment by myugan** [comment]
FYI @walidshaari I delete the emoji for some resources but we can refactor first the repo once done we can add the emoji again.

**Comment by busayyyo** [comment]
Is there a reason why Kim's Udemy course was deleted?

**Comment by myugan** [comment]
> Is there a reason why Kim's Udemy course was deleted?

Have a look at the changes, I move it the resources

**Comment by walidshaari** [comment]
that's many changes in a single commit. against the principle small frequent changes. also, see @busayyyo concern.  let me review it carefully, there should be no conflict merge if you pulled first latest update. why there is a conflict?

**Comment by walidshaari** [comment]
I do not agree with moving CKS training resources to the generic kubernetes security. it is CKS first page, the other page is if it was not specific to CKS but more generic resources for k8s/containers security.

**Comment by myugan** [comment]
okay then @walidshaari let me return it back

File: README.md [hunk 1]
```diff
@@ -27,28 +28,25 @@ Offical exam objectives you review and understand in order to pass the test.
...
-
-  This includes all available language translations of these pages (e.g. https://kubernetes.io/zh/docs)
-  - Tools:
-    - [Trivy documentation](https://github.com/aquasecurity/trivy)
-    - [Sysdig documentation](https://docs.sysdig.com)
-    - [Falco documentation](https://falco.org/docs)
-    - [App Armor documentation](https://gitlab.com/apparmor/apparmor/-/wikis/Documentation)
+## URLs allowed in the extra single tab
+From Chrome or Chromium browser to open one (1) additional tab in order to access Kubernetes Documentation: 
+- https://kubernetes.io/docs and their subdomains
```
Comment (walidshaari) [inline README.md:42]: actually, subdomains here could be wrong as discuss for example is not allowed. my mistake

File: README.md [hunk 2]
```diff
@@ -27,28 +28,25 @@ Offical exam objectives you review and understand in order to pass the test.
...
+- https://github.com/kubernetes and their subdomains
+- https://kubernetes.io/blog and their subdomains
+- [Sysdig documentation](https://docs.sysdig.com)
+- [Falco documentation](https://falco.org/docs)
+- [App Armor documentation](https://gitlab.com/apparmor/apparmor/-/wikis/Documentation)
+
```
Comment (walidshaari) [inline README.md:48]: Trivy dropped?

File: README.md [hunk 3]
```diff
@@ -62,27 +60,22 @@ Offical exam objectives you review and understand in order to pass the test.
...
   - [x] [Slack](#slack)
-  - [x] [Books](#books)
-  - [x] [Youtube Videos](#youtube-videos)
-  - [x] [Webinars](#webinars)
-  - [x] [Containers and Kubernetes Security Training](#containers-and-kubernetes-security-training)
-  - [x] [Extra Kubernetes security resources](generic-kubernetes-containers-security/Kubernetes.md)
+  - [x] [Twitch](#twitch)
+  - [x] [Books](generic-kubernetes-containers-security/Kubernetes.md#books)
+  - [x] [Youtube Videos](generic-kubernetes-containers-security/Kubernetes.md#youtube-videos)
+  - [x] [Containers and Kubernetes Security Training](generic-kubernetes-containers-security/Kubernetes.md#containers-and-kubernetes-security-training)
```
Comment (walidshaari) [inline README.md]: training dropped?

**Review by walidshaari — CHANGES_REQUESTED** [review]
Revet back and use minor updates that we are able or easy to track,  also there are major changes like removing the collapse blocks and icons, these were requested by other users to simplify navigation and focus areas.

**Comment by walidshaari** [comment]
Sorry, @myugan I did a quick review, I am short of time these days. however, I believe for major changes we need to discuss why and how first via issues. and better to incorporated small updates than large ones. e.g. add contributors page. it does not have to be on the main page. so it is easy to track and update

File: README.md (hunk 3 above)
Comment (myugan) [reply README.md]: I think it should be new changes, as the previous one I return it back to main README

**Comment by myugan** [comment]
No problem @walidshaari you can ignore the pull, for now, we can discuss it next time for the updates

**Comment by walidshaari** [comment]
you can refactor it into smaller managable updates

**Pull request closed without merging**